packages/wrangler/src/__tests__/cloudchamber/apply.test.ts

```diff
@@ -51,6 +52,18 @@ function mockCreateApplication(
 	);
 }
 
+function mockAccount(account: CompleteAccountCustomer) {
+	msw.use(
+		http.get(
+			"*/me",
+			async () => {
+				return HttpResponse.json(account);
+			},
+			{ once: true }
+		)
+	);
+}
+
```

Comment:
Can we re-use the `mockAccount` function from `cloudchamber/utils.ts` instead of adding a new function?